query-graphs/style/query-graphs.css

```diff
@@ -6,19 +6,77 @@
   margin: 0 auto;
 }
 
-.qg-tree-label {
-  margin: 0 auto;
+.qg-info-card {
   position: absolute;
   top: 3px;
   right: 3px;
+  background-color: rgba(255, 255, 255, 0.85);
+  border: thin solid #ccc;
+  border-radius: 5px;
+  box-shadow: 2px 2px 2px #888;
+}
+
+.qg-tree-label {
   font-size: 13px;
   font-weight: bold;
   font-family: Monaco, Consolas, monospace;
-  background-color: rgba(255, 255, 255, 0.85);
   padding: 12px;
-  border: thin solid #eee;
+}
+
+.qg-toolbar {
+  float: right;
+  display: inline-block;
+  padding: 2px;
+}
+
+.qg-toolbar-button {
+  position: relative;
+  display: inline-grid;
```

Comment:
`inline-grid` is a pretty heavy-weight/overpowered tool for the simple layout we are trying to achieve here. Also, we aren't really using it to setup any grid anyway.

Afaict, an easier way to achieve the same layout would be:
```
display: inline-block;
vertical-align: bottom;
```